{
  DSLContext create=create(new CustomExceptionListener());
  try {
    create.fetch("invalid sql");
  }
 catch (  E e) {
    assertEquals("ERROR",e.getMessage());
  }
}
